app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1>All Users data will be shown here</h1>
\ No newline at end of file
+<div>
+  <% if @users.empty? %> 
+    <div>There are no users.</div>
+  <% else %>
+    <div>
+      <% @users.each do |user| %>
+        <div>
+          <%= image_tag user.photo, alt: "User pic" %>
+          <%= link_to user_url(user) %>
```

Comment:
- [ ] As shown below in the user index page clicking the link you provided for redirecting to the user show page isn't working so kindly make sure to fix this.
- Here's how it's behaving:
  ![firefox_OkOfevYAL9](https://github.com/Aqurds/coding-thought/assets/104968236/2ace9220-8477-464c-a9e4-febc461e8fcc)

Reply:
Fixed